app/controllers/orders_controller.rb

```diff
@@ -5,7 +5,7 @@ def new
   end
 
   def index
-    
+    @orders = Order.select(params[:user_id]).all
```

Comment:
It might be more secure to use `@orders = Order.select(session[:user_id]).all` or `@orders = current_user.orders`, if it's available.

Reply:
Great idea! Updated to `@orders = current_user.orders`